commit b90b9d572321f8f2102e61965c28930c67f842ec
Author: Leonard Crestez <leonard.crestez@nxp.com>
Date:   Tue Feb 14 17:31:03 2017 +0200

    regulator: Fix regulator_summary for deviceless consumers
    
    commit e42a46b6f52473661ad192f76a128a68fe301df4 upstream.
    
    It is allowed to call regulator_get with a NULL dev argument
    (_regulator_get explicitly checks for it) but this causes an error later
    when printing /sys/kernel/debug/regulator_summary.
    
    Fix this by explicitly handling "deviceless" consumers in the debugfs code.
    
    Signed-off-by: Leonard Crestez <leonard.crestez@nxp.com>
    Signed-off-by: Mark Brown <broonie@kernel.org>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Index: linuxtest/dummy/rpi_1245_b90b9d572321f8f2102e61965c28930c67f842ec.txt
===================================================================
--- /dev/null
+++ linuxtest/dummy/rpi_1245_b90b9d572321f8f2102e61965c28930c67f842ec.txt
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
